tests/integration/beta/test_beta_program.py

```diff
@@ -0,0 +1,48 @@
+from tests.integration.helpers import exec_test_command
+
+BASE_CMD = ["linode-cli", "betas"]
+
+
+def test_beta_list():
+    res = (
+        exec_test_command(BASE_CMD + ["list", "--text", "--delimiter=,"])
+        .stdout.decode()
+        .rstrip()
+    )
+    lines = res.splitlines()
+
+    beta_id = lines[1].split(",")[0]
```

Comment:
There can be a case that in some period of time, there might not be any available beta program. So I think we should skip this line if `len(lines[1].split(",")) == 0` and also skip the view tests below